server/rooms.ts

```diff
@@ -1408,10 +1408,11 @@ export class GlobalRoomState {
 			return;
 		}
 		this.lastReportedCrash = time;
+
 		const stack = err.stack || err.message || err.name || '';
```

Comment:
```suggestion
		const stack = (err && (err.stack || err.message || err.name)) || '';
```

Reply:
when does that happen? ts says that it's not nullable, and I'm inclined to agree based on every obvious call I can see?

Reply:
![image](https://user-images.githubusercontent.com/551184/89524154-eaa67600-d798-11ea-9927-bde5f64c3322.png)

Reply:
This is clearly a bug and no one should ever throw `null` but... this _is_ a crashlogger for bugs.

Reply:
Also TS doesn't say it's not nullable. The only reason TS doesn't complain is because `err` has type `any`, which is currently hugely controversial anyway.

Reply:
fun!